NEWS.md

```diff
@@ -31,6 +31,9 @@ for human readability.
 
 #### Removed
 
+- Migrate neural network-based shock indicators to a new respository
```

Comment:
```suggestion
- Migrate neural network-based shock indicators to a new repository
```

Reply:
This should also go to the section of breaking changes when updating from v0.5 to v0.6 of Trixi.jl

Reply:
Fixed in 7a7871066.